extensions/amp-subscriptions/0.1/entitlement.js

```diff
@@ -45,29 +43,24 @@ export class Entitlement {
    * @param {string} [input.source]
    * @param {string} [input.raw]
    * @param {string} [input.service]
-   * @param {!Array<string>} [input.products]
-   * @param {?string} [input.subscriptionToken]
-   * @param {boolean} [input.loggedIn]
-   * @param {?MeteringData} [input.metering]
+   * @param {boolean} [input.granted]
```

Comment:
In this case we should also update amp-subscriptions-google platform in this PR, right?